web-server/src/components/AnimatedInputWrapper/AnimatedInputWrapper.tsx

```diff
@@ -0,0 +1,119 @@
+import { themeColors } from "@/theme/schemes/theme";
+
+const AnimatedInputWrapper = () => {
+  const [isAnimationVisible, setIsAnimationVisible] = useState(true);
+  const [value, setValue] = useState("");
+
+  const addAnimation = useCallback(
+    () => setIsAnimationVisible(true),
+    [setIsAnimationVisible]
+  );
+
+  const handleChange = useCallback(
+    (e: any) => setValue(e.target.value),
+    [setValue]
+  );
+
+  return (
+    <div>
+        {isAnimationVisible && !value && <AnimatedPlaceHolderWrapper />}
+        <input
+          value={value}
+          onChange={handleChange}
+          onBlur={addAnimation}
+          placeholder={!isAnimationVisible ? "Search" : ""}
+          style={{backgroundColor: themeColors.primaryAlt,border: "none",outline: "none", right: "0px"}}
+        />
+    </div>
+    
+  );
+};
+
+const popRepos = ["golang/go", "mozilla/rust", "apple/swift", "oven-sh/bun"];
+
+const AnimatedPlaceHolderWrapper = () => {
+  return (
+    <div
+      style={{
+        position: "absolute",
+        top: "50%",
+        transform: "translate(0%,-50%)",
+        fontSize: "1em",
+        display: "flex",
+        gap: "4px",
+      }}
+    >
+      <span>Search for</span>
+      <AnimatedRepos />
+    </div>
+  );
+};
+
+const AnimatedRepos = () => {
+  const [animationIndex, setAnimationIndex] = useState(0);
+  const [repos, setRepos] = useState(popRepos);
+
+  const increment = () =>
+    setAnimationIndex((prev) => {
+      if (prev >= popRepos.length - 1) return 0;
+      return prev + 1;
+    });
+
+  const rearrangeArray = () => {
+    const beforeIndex = repos.slice(0, 1);
+    const fromIndexOnwards = repos.slice(1);
+    const arr = fromIndexOnwards.concat(beforeIndex);
+    setRepos(arr);
+  };
+  
+  useEffect(() => {
+  const animationInterval = setInterval(increment, 2000);
+  let repoInterval: NodeJS.Timeout | undefined;
+  const timeout = setTimeout(() => {
+    repoInterval = setInterval(rearrangeArray, 2000);
+  }, 500);
+  return () => {
+    clearInterval(animationInterval);
+    if (repoInterval !== undefined) {
+      clearInterval(repoInterval);
+    }
+    clearTimeout(timeout);
+  };
+}, []);
+
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        overflow: "hidden",
+        height: "1.4em",
+        alignItems: "start",
+      }}
+    >
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          alignItems: "start",
+          position: "relative",
+          bottom: animationIndex * 1.4 + "em",
+          transition: "all 0.5s ease",
+        }}
+      >
+        {repos.map((item) => (
+          <div
+            key={item}
+            style={{
+              height: "1.4em",
+            }}
+          >
+            <span>{item}</span>
+          </div>
+        ))}
+      </div>
+    </div>
+  );
+};
+export default AnimatedInputWrapper;
```

Comment:
I think the linter didn't run properly again.
Could you try running the command I shared last time?

Or let me know if you're unsure about which command to run.

Reply:
This isn't the only instance of the linter having failed, in this PR.

If we don't have an issue created for this, now is a time as good as any, @samad-yar-khan @e-for-eshaan @adnanhashmi09

Reply:
Can you tell me about which command to run?